gtest_parallel_tests.py

```diff
@@ -143,6 +159,35 @@ def test_run_task_basic(self):
 
     self.assertEqual(task_manager.global_exit_code, 1)
 
+  @unittest.skipIf(sys.platform == 'win32', 'non-Windows specific')
```

Comment:
Ok, I see. But then mocking out just sys.platform is not enough. One has to mock os.path.expanduser too.

> I don't expect anyone to run these tests on Windows

I don't know. There are a lot of people work on Windows. As for my case we definitely will run gtes-parallel on Windows. So, Windows specific code will definitely run.

> Does that make sense?

This will definitely increase coverage but maintaining such tests will be more difficult. I'll see what I can do.